docs/pages/web/textfield.js

```diff
@@ -700,6 +698,78 @@ function TextFieldExample() {
     />
   );
 }
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Maximum length"
+          columns={2}
+          description={`Textfield supports the native [maxlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength) input attribute. \`maxLength\` sets the maximum number of characters allowed to be entered by the user in Textfield.
+          
+For controlled TextFields, if the initial value in the external state exceeds the \`maxLength\` value, Textfield will display the long value; however, the user won't be able to add additional characters and they will only be allowed to delete characters. 
```

Comment:
Not sure what "Textfield will display the long value" means... 

maybe "Textfield will display the full value"

Reply:
DONE